.semaphore/semaphore.yml

```diff
@@ -4,33 +4,27 @@ agent:
   machine:
     type: e1-standard-2
     os_image: ubuntu1804
-
+global_job_config:
+  prologue:
+    commands:
+      - sem-version python 3.8
+      - sh ~/get-sam-cli.sh
+      - checkout
+      - cache restore requirements-$SEMAPHORE_GIT_BRANCH-$(checksum requirements.txt)
```

Comment:
SHOULD:  I don't think we should optimize our file structure for semaphore. To me, this really is a `requirements-dev.txt` because the libraries are not packaged or deployed anywhere.